### PR TITLE
SER-4190 UCP: Issue with user rename process

### PR DESCRIPTION
**Pull request by mmeller-wikia**
http://wikia-inc.atlassian.net/browse/SER-4190

### CODE REVIEW

**Review by lomiarz — APPROVED** [review]

File: lib/Wikia/src/Tasks/Tasks/RenameUserPagesTask.php [hunk 1]
```diff
@@ -156,8 +156,8 @@ public function renameLocalPages( string $oldUserName, string $newUserName ) {
 
 		$this->renameCheckUserMentions( $oldUserName, $newUserName );
 
-		$user = User::newFromName( $newUserName );
-		$user->deleteCache();
+		User::newFromName( $newUserName )->deleteCache();
+		User::newFromName( $oldUserName )->deleteCache();
```
Comment (slayful) [inline lib/Wikia/src/Tasks/Tasks/RenameUserPagesTask.php:160]: Why these changes?
